docs/index.md

```diff
@@ -96,6 +126,30 @@ class TestCrud(unittest.TestCase):
 		user = self.your_core_lib.user.create({User.name.key: 'John Dow'})
     self.assertDictEqual(user, self.your_core_lib.user.get(user[User.id.key]))
 ```
+#### Code Explained:
+<b>Writing unit tests for your YourCoreLib class using the unittest framework, alongside Hydra for configuration management.</b>
+- <b>Importing necessary modules:</b>
+
+  - <b>unittest:</b> Standard Python module for writing unit tests.
+  - <b>hydra:</b> Hydra library for managing configurations.
+  - <b>GlobalHydra:</b> A class from Hydra used for global configuration management.
+- <b>Defining a function get_config():</b>
```

Comment:
Defining a function get_config() to use Hydra for loading testing config.yaml located under the testing folder.
The config.yaml contain values specific for the testing environment

make this shorter